json-rpc/eth_getBalance.md

```diff
@@ -0,0 +1,17 @@
+# `Eth_getBalance`
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [RFC-2119](https://www.ietf.org/rfc/rfc2119.txt).
+Specification | Description 
+---|---
+1|Eth_getBalance must be sent a valid Ethereum address as the first parameter|
+1.1 |Eth_getBalance should return error code -32602 "invalid argument 0: json: cannot unmarshal hex string of odd length into Go value of type common.Address" if the address send has an odd length|
+1.2 |Eth_getBalance should return error code -32602 "invalid argument 0: hex string has length n, want 40 for common.Address" if address length is even but not equal to 40|
+1.3 |Eth_getBalance should return error code -32602  "invalid argument 0: json: cannot unmarshal hex string without 0x prefix into Go value of type common.Address" if the address is blank or a random string|
+2 |Eth_getBalnace must be sent the block that you want the balance from as the second parameter|
```

Comment:
this is very inefficient way of describing JSON RPC, it seems. Parameters and return values should be described in the EIP-1474 format while edge cases as a list of edge cases